### DISCUSSION TITLE
Can you run the Minio Console in an iframe?

### DISCUSSION THREAD

**Discussion by humphaz** [minio/minio#16529] [Q&A] [answered]
Hi,

I'm trying to use the console in an iframe, is this possible?

I know for retrieving objects within the object stores, e.g. graphics, there is functionality to allow that as long as STS authentication is used.

But can you use the actual console itself in an iframe, if so, must this be done in a similar way by using a 3rd party identity service like LDAP?

Thanks in advance,

Glyn

**Comment by harshavardhana** [comment, marked as answer]
You can look at the code in operator console on how to do it.
